src/commands/interviews/interviewerDomain.ts

```diff
@@ -0,0 +1,48 @@
+import { Message } from 'discord.js';
+import { CommandoClient, CommandoMessage } from 'discord.js-commando';
+import { BaseCommand } from '../../utils/commands';
+import { availableDomains, getAvailableDomainsString, getInterviewer, toggleDomain } from '../../components/interview';
+import { parseDomainArg, validateDomainArg } from './utils';
+
+class InterviewerDomainCommand extends BaseCommand {
+  constructor(client: CommandoClient) {
+    super(client, {
+      name: 'interviewer-domain',
+      group: 'interviews',
+      memberName: 'domain',
+      args: [
+        {
+          key: 'domain',
+          prompt: `enter one of ${getAvailableDomainsString()}.`,
+          type: 'string',
+          validate: validateDomainArg,
+          parse: parseDomainArg
+        }
+      ],
+      description: 'Adds or removes a domain from your interviewer profile.',
+      examples: ['.interviewer-domain frontend', '.interviewerdomain pm']
+    });
+  }
+
+  async onRun(message: CommandoMessage, args: { domain: string }): Promise<Message> {
+    const { domain } = args;
+    const { id } = message.author;
+
+    //check user signed up to be an interviewer
+    if (!(await getInterviewer(id))) {
```

Comment:
Whenever u kinda do a null check, or '?' conditional, I think it'd be good to try to be consistent in all the files but im just being extra cause idk what egg to find lol

Reply:
do you mean if statement vs ternary statement? I used if statement here because we only care about the case where there is not interviewer with that ID

Reply:
mb lemme clarify - here u have if `!func()`
usually I saw 
`bool isCool = func();`
`if isCool do smth`

Reply:
I think usually we do that if we need the `isCool` variable later on, in this case we don't need the interviewer